src/webgpu/util/conversion.ts

```diff
@@ -762,37 +766,44 @@ function valueFromBytes(workingDataOut: TypedArrayBufferView, buf: Uint8Array, o
   return workingDataOut[0];
 }
 
+export const TypeAbstractInt = new ScalarType(
+  'abstract-int',
+  8,
+  (buf: Uint8Array, offset: number) =>
+    abstractInt(valueFromBytes(workingDataF64, buf, offset) as bigint)
```

Comment:
Why `workingDataF64`? It just happens to be the right byte length?

Reply:
That is a copy-pasta mistake.